File: DayZPromo/src/types/PromoCode.ts
export interface PromoCode {
    id: string;
    name: string;
    promo: string;
    expirationDate: string;
    qrCodePath?: string;
    finalImagePath?: string;
    isActive: boolean;
}

export interface PostImageDto {
    name: string;
    expirationDate: string; // Всегда строка в формате ISO
} 

File: CrazyDayZ.Promo/Models/PromoCode.cs
namespace CrazyDayZ.Promo.Models
{
    public class PromoCode
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Promo { get; set; }
        public string ExpirationDate { get; set; }
        public string QrCodePath { get; set; }
        public string FinalImagePath { get; set; }
    }
} 